UseMultiArch.cmake

```diff
@@ -20,7 +20,7 @@ if ("${CMAKE_SYSTEM_NAME}" MATCHES "Linux" AND
   if (EXISTS "/etc/debian_version")
 	set (_libdir_def "lib/${CMAKE_LIBRARY_ARCHITECTURE}")
 	set (_libdir_noarch "lib")
-  else (EXISTS "/etc/debian_version")
+  elseif (EXISTS "/etc/fedora-release")
```

Comment:
please add OR EXISTS /etc/redhat-release

afair suse should fall in this category as well, but I don't have any suse around to check.